source/numeric/SVInt.cpp

```diff
@@ -1685,7 +1726,7 @@ SVInt& SVInt::assignSlowCase(const SVInt& rhs) {
 
 logic_t SVInt::equalsSlowCase(const SVInt& rhs) const {
     if (unknownFlag || rhs.unknownFlag)
```

Comment:
Can you add a comment saying that if there are unknowns we can't know whether the numbers are equal, but we might be able to know if they are definitely not equal, which is what the xor + reductionOr is doing.

Reply:
Yes, I add
```c++
    if (unknownFlag || rhs.unknownFlag) {
        // We can't know whether the numbers are definitely equal, but if there is a 0/1 pair, it is
        // definitely not equal. xor detects 0/1 pairs for each bit and !reductionOr collects all
        // pairs.
        return !(*this ^ rhs).reductionOr();
    }
```